SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -78,6 +78,11 @@ bool FSpatialGDKEditor::GenerateSchema(bool bFullScan)
 
 	TryLoadExistingSchemaDatabase();
 
+	if (bFullScan)
+	{
+		DeleteGeneratedSchemaFiles();
```

Comment:
Please add a comment about why this is necessary

Reply:
It isn't, removed.